near-sdk/src/environment/mock/mocked_blockchain.rs

```diff
@@ -126,6 +136,51 @@ fn sdk_context_to_vm_context(context: VMContext) -> near_vm_logic::VMContext {
     }
 }
 
+fn action_to_sdk_action(action: &PrimitivesAction) -> VmAction {
+    match action {
+        PrimitivesAction::CreateAccount(_) => VmAction::CreateAccount,
+        PrimitivesAction::DeployContract(c) => VmAction::DeployContract { code: c.code.clone() },
+        PrimitivesAction::FunctionCall(f) => VmAction::FunctionCall {
+            function_name: f.method_name.clone(),
+            args: f.args.clone(),
+            gas: Gas(f.gas),
+            deposit: f.deposit,
+        },
+        PrimitivesAction::Transfer(t) => VmAction::Transfer { deposit: t.deposit },
+        PrimitivesAction::Stake(s) => {
+            VmAction::Stake { stake: s.stake, public_key: pub_key_conversion(&s.public_key) }
+        }
+        PrimitivesAction::AddKey(k) => match &k.access_key.permission {
+            near_primitives::account::AccessKeyPermission::FunctionCall(f) => {
+                VmAction::AddKeyWithFunctionCall {
+                    public_key: pub_key_conversion(&k.public_key),
+                    nonce: k.access_key.nonce,
+                    allowance: f.allowance,
+                    receiver_id: f.receiver_id.parse().unwrap(),
+                    function_names: f.method_names.clone(),
+                }
+            }
+            near_primitives::account::AccessKeyPermission::FullAccess => {
+                VmAction::AddKeyWithFullAccess {
+                    public_key: pub_key_conversion(&k.public_key),
+                    nonce: k.access_key.nonce,
+                }
+            }
+        },
+        PrimitivesAction::DeleteKey(k) => {
+            VmAction::DeleteKey { public_key: pub_key_conversion(&k.public_key) }
+        }
+        PrimitivesAction::DeleteAccount(a) => {
+            VmAction::DeleteAccount { beneficiary_id: a.beneficiary_id.parse().unwrap() }
+        }
```

Comment:
Very small nit: any reason why you didn't inline these like you have done with others above? Sorry for nitpicking, this is just triggering my OCD :)

Reply:
it's just default formatting, the line is too long and it pushes it like this